std_msgs/Float64MultiArray  state
---
geometry_msgs/Pose          end_effector